### PR TITLE
PATRON2020 - 261 Gatevay - cookies bug

### PR DESCRIPTION
**Pull request by kuba8686**

### CODE REVIEW

File: gateway/gateway.cpp [hunk 1]
```diff
@@ -53,18 +53,20 @@ struct DashboardMiddleware
...
-        if (cookiectx.get_cookie(COOKIE_NAME) != COOKIE_VALUE)
+        if (cookiectx.get_cookie(COOKIE_NAME) != COOKIE_VALUE) {
             res = crow::response(403);
+            res.end();
+        }
         else
         {
             CROW_LOG_INFO << "COOKIE OK";
         }
+
```
Comment (Siwy160) [inline gateway/gateway.cpp:64]: blank line :)

**Comment by Siwy160** [comment]
Short explanation what was wrong and how actually your change fixed it would be nice.

**Comment by kuba8686** [comment]
> 
> 
> Short explanation what was wrong and how actually your change fixed it would be nice.

Addet comment to JIRA ticket
https://tracker.intive.com/jira/browse/PATRON2020-261

File: gateway/gateway.cpp (hunk 1 above)
Comment (kuba8686) [reply gateway/gateway.cpp:64]: done

**Pull request merged**